ape_aws/kms/_cli.py

```diff
@@ -124,30 +113,31 @@ def create_key(
 def import_key(
     cli_ctx,
     alias_name: str,
-    private_key: bytes | str,
+    private_key_path: Path,
     administrators: list[str],
     users: list[str],
     description: str,
-    import_from_file: bool,
-    file_path: str | Path,
     import_from_mnemonic: bool,
     hd_path: str,
 ):
-    if import_from_file:
-        if isinstance(file_path, str):
-            path = Path(private_key)
-        if path.exists() and path.is_file():
-            cli_ctx.logger.info(f"Reading private key from {path}")
-            private_key = path.read_text().strip()
-
-    if import_from_mnemonic:
+    if private_key_path:
+        if isinstance(private_key_path, str):
+            private_key_path = Path(private_key_path)
+        if private_key_path.exists() and private_key_path.is_file():
+            cli_ctx.logger.info(f"Reading private key from {private_key_path}")
+            private_key = private_key_path.read_text().strip()
+
+    elif import_from_mnemonic:
         if not hd_path:
             hd_path = ETHEREUM_DEFAULT_PATH
         mnemonic = click.prompt("Enter your mnemonic phrase", hide_input=True)
         EthAccount.enable_unaudited_hdwallet_features()
         account = EthAccount.from_mnemonic(mnemonic, account_path=hd_path)
         private_key = account.key.hex()
 
+    else:
+        private_key = input("Enter your private key: ")
```

Comment:
Click has https://click.palletsprojects.com/en/8.1.x/api/#click.prompt (with `hide_input=True`) for this scenario